Uninstall-BitbucketServer.ps1

```diff
@@ -12,26 +12,41 @@
 
 [CmdletBinding()]
 param(
-    [Parameter(Mandatory=$true)]
     [string]
-    $BitbucketInstallationPath,
+    $BitbucketInstallationPath = (Join-Path -Path $env:SystemDrive -ChildPath 'Atlassian\Bitbucket'),
 
-    [Parameter(Mandatory=$true)]
     [string]
-    $BitbucketHomePath
+    $BitbucketApplicationDataPath = (Join-Path -Path $env:SystemDrive -ChildPath 'Atlassian\ApplicationData\Bitbucket')
 )
 
 Set-StrictMode -Version 'Latest'
+#Requires -RunAsAdministrator
 
-& (Join-Path -Path $PSScriptRoot -ChildPath '.\Carbon\Import-Carbon.ps1' -Resolve)
+& {
+    $VerbosePreference = 'SilentlyContinue'
 
-Get-Service -Name '*Bitbucket*' | 
+    $importCarbonModulePath = (Join-Path -Path $PSScriptRoot -ChildPath '.\PSModules\Carbon\2.6.0\Import-Carbon.ps1')
+    if (-not (Test-Path -Path $importCarbonModulePath -PathType Leaf))
+    {
+        Save-Module -Name 'Carbon' -Path (Join-Path -Path $PSScriptRoot -ChildPath 'PSModules') -RequiredVersion '2.6.0' -Force

```

Comment:
Replace this with a message to run `init.ps1` to install Carbon.

Reply:
Did you mean `build.ps1`? The init script doesn't install Carbon, the Whiskey build does.

Reply:
Yes. `build.ps1 -Initialize`.